From 3cca5450c808d04e23b30cb99fcc9f3fa7f62832 Mon Sep 17 00:00:00 2001
From: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
Date: Thu, 23 Sep 2021 21:32:40 +0530
Subject: [PATCH] Fix vendor sepolicy declaration

vendor defined properties should start with vendor.**
sys.display.size needs to be vendor.sys.display.size

Tracked-On: NA
Signed-off-by: Tanuj Tekriwal <tanuj.tekriwal@intel.com>
---
 aafd/property_contexts | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/aafd/property_contexts b/aafd/property_contexts
index ff31185..718f834 100644
--- a/aafd/property_contexts
+++ b/aafd/property_contexts
@@ -6,4 +6,4 @@ vendor.suspend u:object_r:vendor_suspend_prop:s0
 vendor.usb.controller u:object_r:vendor_usb_controller_prop:s0
 vendor.power.fixed_performance_scale_factor         u:object_r:vendor_fixed_perf_prop:s0
 vendor.mount.ep0 u:object_r:vendor_mount_ep0_prop:s0
-sys.display.size u:object_r:vendor_display_prop:s0
+vendor.sys.display.size u:object_r:vendor_display_prop:s0
-- 
2.17.1

